manual/manual/tutorials/objectexamples.etex

```diff
@@ -2,10 +2,35 @@
 \label{c:objectexamples}
 \pdfchapterfold{-15}{Tutorial: Objects in OCaml}
 %HEVEA\cutname{objectexamples.html}
-{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
+{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
 
 \bigskip
 
+TODO:
+>> To me, this section seems to be showing how powerful the type system is and
+how you can get all kinds of different behavior for subclassing, self, etc.
+It’s too much.  I would suggest taking examples from Java and C++ classes/objects
+and showing how to create equivalent constructs in OCaml—more of a cookbook approach.
+It seems to me there are a huge number of things you can do with the type system,
+of which only a few are really useful.  If you’re writing a program with classes
+and objects, you probably want them all to have similar behavior with regards to
+subclassing, self, etc. just for your own sanity.
```

Comment:
No doubt--I'm an OCaml beginner.  It would be really helpful to have some text in the document that explains this.  I don't know enough to write such an explanation.

Reply:
After rereading the introduction of the object chapter, I think that its introductive paragraph needs to be reworked to avoid ominous but not informative warnings to C++/Java developpers.